File: lib_common/src/main/java/com/fly/tour/common/util/ToastUtil.java
package com.zjx.vcars.common.util;

import android.widget.Toast;

import com.zjx.vcars.common.BaseApplication;

/**
 * Description: <吐司工具类><br>
 * Author: gxl<br>
 * Date: 2018/6/11<br>
 * Version: V1.0.0<br>
 * Update: <br>
 */
public class ToastUtil {

    public static void showToast(String message) {
        Toast.makeText(BaseApplication.getInstance(), message, Toast.LENGTH_SHORT).show();
    }

    public static void showToast(int resid) {
        Toast.makeText(BaseApplication.getInstance(), BaseApplication.getInstance().getString(resid), Toast.LENGTH_SHORT)
                .show();
    }
}

File: lib_common/src/main/java/com/fly/tour/common/util/ToastUtil.kt
package com.fly.tour.common.util

import android.widget.Toast

import com.fly.tour.common.BaseApplication

/**
 * Description: <吐司工具类><br>
 * Author: mxdl<br>
 * Date: 2018/6/11<br>
 * Version: V1.0.0<br>
 * Update: <br>
</吐司工具类> */
object ToastUtil {

    fun showToast(message: String) {
        Toast.makeText(BaseApplication.instance, message, Toast.LENGTH_SHORT).show()
    }

    fun showToast(resid: Int) {
        Toast.makeText(BaseApplication.instance, BaseApplication.instance?.getString(resid), Toast.LENGTH_SHORT)
                .show()
    }
}